plugins/SkyCultureMaker/src/gui/scmSkyCultureDialog.ui

```diff
@@ -234,16 +234,230 @@ p, li { white-space: pre-wrap; }
+                  <string>Constellations:</string>
+                  </property>
+                </widget>
+                </item>
+                <item>
+                <widget class="QTextEdit" name="constellationsDescTE"/>
+                </item>
+
+                <!-- Milky Way -->
+                <item>
+                <widget class="QLabel" name="milkyWayLbl">
+                  <property name="text">
+                  <string>Milky Way:</string>
+                  </property>
+                </widget>
+                </item>
+                <item>
+                <widget class="QTextEdit" name="milkyWayTE"/>
+                </item>
+
+                <!-- Other Celestial Objects -->
+                <item>
+                <widget class="QLabel" name="otherObjectsLbl">
+                  <property name="text">
+                  <string>Other Celestial Objects:</string>
+                  </property>
+                </widget>
+                </item>
+                <item>
+                <widget class="QTextEdit" name="otherObjectsTE"/>
+                </item>
+
+                <!-- About -->
+                <item>
+                <widget class="QLabel" name="aboutLbl">
+                  <property name="text">
+                  <string>About:</string>
+                  </property>
+                </widget>
+                </item>
+                <item>
+                <widget class="QTextEdit" name="aboutTE"/>
+                </item>
+
+                <!-- Authors -->
+                <item>
+                <widget class="QLabel" name="authorsLbl">
+                  <property name="text">
+                  <string>Authors:</string>
+                  </property>
+                </widget>
+                </item>
+                <item>
+                <widget class="QTextEdit" name="authorsTE"/>
+                </item>
+
+                <!-- Acknowledgements -->
+                <item>
+                <widget class="QLabel" name="acknowledgementsLbl">
+                  <property name="text">
+                  <string>Acknowledgements:</string>
+                  </property>
+                </widget>
+                </item>
+                <item>
+                <widget class="QTextEdit" name="acknowledgementsTE"/>
+                </item>
+
+                <!-- References -->
+                <item>
+                <widget class="QLabel" name="referencesLbl">
+                  <property name="text">
+                  <string>References:</string>
+                  </property>
+                </widget>
+                </item>
+                <item>
+                <widget class="QTextEdit" name="referencesTE"/>
+                </item>
+
+                <!-- Classification -->
+                <item>
+                  <widget class="QLabel" name="classificationLbl">
+                    <property name="font">
+                    <font>
+                      <pointsize>14</pointsize>
```

Comment:
The "Classification of the Sky Culture" has a font size of 14, while all other labels are implicitly 13 it seems.